src/selectors/profile.js

```diff
@@ -827,3 +828,21 @@ export const getContainsPrivateBrowsingInformation: Selector<boolean> =
 
     return hasPrivateThreads;
   });
+
+/**
+ * Returns the TIDs of the threads that are profiled.
+ */
+export const getProfiledThreadTids: Selector<Set<Tid>> = createSelector(
```

Comment:
Optional nit: I'd use `getProfiledThreadIds`. Indeed the `T` in `Tid` means `Thread` already, so in `Thread Tid` the `T` is redundant.